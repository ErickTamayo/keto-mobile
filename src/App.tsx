import * as React from 'react'
import { StyleSheet, Text, View } from 'react-native'

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
})
export default class App extends React.Component {
  renderWorld = (): string => 'World'

  public render(): JSX.Element {
    return (
      <View style={styles.container}>
        <Text>Hello {this.renderWorld()}</Text>
      </View>
    )
  }
}
