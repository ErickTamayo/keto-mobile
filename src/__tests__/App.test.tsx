import 'react-native'
import React from 'react'
import App from '../App'
import { shallow } from 'enzyme'

describe('<App />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
