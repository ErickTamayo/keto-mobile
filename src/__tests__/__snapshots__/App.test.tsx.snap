// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Component>
    Hello World!
  </Component>
</Component>
`;
